docs/ossm/README.md

```diff
@@ -0,0 +1,196 @@
+[Return to Project Root](../../)
+
+# Table of Contents
+
+- [User Documentation](#user-documentation)
+- [istio-csr Integration](#istio-csr-integration)
+  - [Common Setup](#common-setup)
+  - [InPlace Strategy](#inplace-strategy)
+  - [RevisionBased Strategy](#revisionbased-strategy)
+  - [Verify](#verify)
+  - [RevisionBased Upgrades](#upgrades---revisionbased)
+
+# User Documentation
+
+This documentation is specific to the OpenShift Service Mesh product and may differ slightly from the [Sail Operator documentation](../README.md). If you are using the OpenShift Service Mesh 3 operator, you should consider this documentation the source of truth. If there is not documentation here for a particular topic, follow the [Sail Operator documentation](../README.md).
+
+# istio-csr Integration
```

Comment:
I think this would be better called "Cert-manager and istio-csr integration" to capture that cert-manager is the goal and istio-csr is the intermediary.